themes/digital.gov/src/scss/new/_topics-button.scss

```diff
@@ -52,27 +54,30 @@
     font-size: font-size("sans", "2xs");
     height: fit-content;
     margin: auto;
-    padding-bottom: units("05");
-    padding-top: units("05");
+    padding-left: units("05");
```

Comment:
updated to left and right to fix the visual error that came up the circle around the numbers